connector_dns/unit/binder.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Elico Corp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from datetime import datetime
+from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from openerp.addons.connector.connector import Binder
+from ..backend import dns
+
+
+class DNSPodBinder(Binder):
```

Comment:
Is this parent class necessary? I haven't seen this in the other connectors.

Also, naming looks off.